docker/jenkins.yml

```diff
@@ -48,10 +48,10 @@ jenkins:
       adminUserNames: "MancunianSam,Dclipsham,ian-hoyle,suzannehamilton,TomJKing,Lholbourn,techncl"
```

Comment:
Should we keep Ian on this list?

Reply:
Yeah, may as well. I've taken him off the list.